lib/onelogin/ruby-saml/response.rb

```diff
@@ -368,6 +368,9 @@ def validate(collect_errors = false)
 
         if collect_errors
           validations.each { |validation| send(validation) }
+          unless @errors.empty?
+            Rails.logger.warn("SAML ERRORS: #{@errors}")
+          end
```

Comment:
Just as a sanity check -- you tested this locally and it worked? I'm a little surprised that `ruby-saml` knows about Rails, but that's the new ES6-me-that-loves-explicit-imports talking.